MapboxNavigation/DataCache.swift

```diff
@@ -49,7 +49,7 @@ public class DataCache: NSObject, BimodalDataCache {
     /*
      Clears out the memory cache.
      */
-    public func clearMemory() {
+    @objc public func clearMemory() {
```

Comment:
Why did this method gain an `@objc` that it didn’t already have?

Reply:
https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/DataCache.swift#L14